mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -7171,6 +7195,34 @@ resource "google_container_cluster" "primary" {
 `, cluster, np)
 }
 
+func testAccContainerCluster_withEnableKubernetesBetaAPIs(cluster, np string) string {
+	return fmt.Sprintf(`
+data "google_container_engine_versions" "uscentral1a" {
+  location = "us-central1-a"
+}
+
+resource "google_container_cluster" "primary" {
+  name     = "%s"
+  location = "us-central1-a"
+
+  min_master_version = data.google_container_engine_versions.uscentral1a.release_channel_latest_version["STABLE"]
+  # TODO: Migrate to supported Beta APIs when available
```

Comment:
I'm not sure I understand the action to be taken from this - could you add more information to the comment? And/or we could discuss in this thread.

Reply:
Added more context.